bigbluebutton-html5/imports/ui/components/audio/audio-modal/container.jsx

```diff
@@ -8,7 +8,11 @@ const AudioModalContainer = props => <AudioModal {...props} />;
 
 const APP_CONFIG = Meteor.settings.public.app;
 
-const { listenOnlyMode, forceListenOnly, skipCheck } = APP_CONFIG;
+const { listenOnlyMode, forceListenOnly } = APP_CONFIG;
+
+// we always skip Echo test on mobile, regardless of the config
+const skipCheck = window.matchMedia('only screen and (max-width: 40em)').matches ?
```

Comment:
`window.matchMedia('only screen and (max-width: 40em)').matches || APP_CONFIG.skipCheck`

Reply:
Also, is this query the safest way to match for a mobile device? I first thought you'd be matching OS...

Reply:
I think there are other places in the code with the same type of "Is this a phone" checks. I think it was maybe @KDSBrowne who put a check in before. Might make sense to make it a utility function and reuse the functionality.

Reply:
There is a utility for that now in `/imports/utils/deviceInfo.js`

Reply:
In the context of what @antobinary is asking, I don't think this is safe either: https://github.com/bigbluebutton/bigbluebutton/blob/v2.0.x-release/bigbluebutton-html5/imports/utils/deviceInfo.js#L12.
Do you guys think we should use the `mobile` and `tablet` flags from `bowser`. We already use `bowser` here: https://github.com/bigbluebutton/bigbluebutton/blob/v2.0.x-release/bigbluebutton-html5/imports/ui/components/audio/permissions-overlay/component.jsx#L39.

Reply:
No matter what solution is chosen it should be in a utility function because the question is the same and we don't want duplicated functionality all over our code base.